styles/godoc.less

```diff
@@ -25,24 +19,38 @@
   &::before {
     content: "";
     position: absolute;
-    top: -4px;
-    left: 0;
-    border: 4px solid;
-    border-color: transparent transparent @godoc-bg @godoc-bg;
+    top: -@godoc-border-width * 3;
+    left: -@godoc-border-width;
+    border: @godoc-border-width * 1.5 solid;
+    border-color: transparent transparent @overlay-border-color @overlay-border-color;
+  }
+
+  .close {
+    position: absolute;
+    top: @component-padding/2;
+    right: @component-padding/2;
+    color: lighten(@text-color, 50%);
+    text-shadow: none;
+    &::before {
+      margin-right: 0;
+    }
   }
 }
 
-.close {
-  color: lighten(@text-color, 50%);
-  text-shadow: none;
+#godoc-message {
+  // overflow: auto; // TODO: still not getting scroll bars...
```

Comment:
The amount of content totally depends on what the user gets doc for.  Generally pretty short for a function, but for a package it could be quite long.

I know what the problem is here, I just don't know how to fix it.

Right now, the size of the `div` that gets created is always big enough to fit the text, so it doesn't display scrollbars because it doesn't think they're needed.  Even if there's a ton of text, it just makes the `div` thousands of pixels tall (even though most of it is off screen).  If I set `max-height` to something small I do get scrollbars.

This isn't a great solution - it requires me to pick some arbitrary height, and it still behaves funky if the tooltip is displayed near the bottom of the editor.